File: 1693-sum-of-all-odd-length-subarrays/1693-sum-of-all-odd-length-subarrays.py
class Solution(object):
    def sumOddLengthSubarrays(self, arr):
        """
        :type arr: List[int]
        :rtype: int
        """
        sum1=0

        for i in range(len(arr)):
            for j in range(i,len(arr),2):
                sum1+=sum(arr[i:j+1])  
        return sum1
               

File: 1693-sum-of-all-odd-length-subarrays/1693-sum-of-all-odd-length-subarrays.c
int sumOddLengthSubarrays(int* arr, int arrSize) {
   int start,end,sum=0,length,i;

   for(start=0;start<arrSize;start++)
   {
        for(end=start;end<arrSize;end++)
        {
            length=end-start+1;
            if(length%2==1)
            {
                for(i=start;i<=end;i++)
                {
                    sum+=arr[i];
                }
            }
        }
   }
    
   return sum;
}